index.js

```diff
@@ -502,6 +506,455 @@
   }
   S.is = def('is', {}, [TypeRep(a), $.Any, $.Boolean], is);
 
+  //. ### Showable
+
+  //# toString :: Any -> String
+  //.
+  //. Alias of [`Z.toString`][].
+  //.
+  //. ```javascript
+  //. > S.toString(-0)
+  //. '-0'
+  //.
+  //. > S.toString(['foo', 'bar', 'baz'])
+  //. '["foo", "bar", "baz"]'
+  //.
+  //. > S.toString({x: 1, y: 2, z: 3})
+  //. '{"x": 1, "y": 2, "z": 3}'
+  //.
+  //. > S.toString(S.Left(S.Right(S.Just(S.Nothing))))
+  //. 'Left(Right(Just(Nothing)))'
+  //. ```
+  S.toString = def('toString', {}, [$.Any, $.String], Z.toString);
+
+  //. ### Fantasy Land
+  //.
+  //. Sanctuary is compatible with the [Fantasy Land specification][FL].
+
+  //# equals :: a -> b -> Boolean
+  //.
+  //. Curried version of [`Z.equals`][].
+  //.
+  //. ```javascript
+  //. > S.equals(0, -0)
+  //. false
+  //.
+  //. > S.equals(NaN, NaN)
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 3]))
+  //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just(['1', '2', '3']))
+  //. false
+  //. ```
+  S.equals = def('equals', {}, [a, b, $.Boolean], Z.equals);
```

Comment:
I think in "your" codebase where you know types of values, you will only use equals when two values have same type, as if you know that one value has different type then the other, it's useless to use equals as it will return false for every value of those types.

But one might use equals in library like code, when someone outside of the library could pass some value of any type. But for cases like this one could just use S.unchecked.equals.

So after writing/thinking on it i think `equals:: a -> a -> Boolena` would be nice if one could also do this `S.unchecked.equals` to access unchecked version of `equals`